src/main/resources/lang/default.lang

```diff
@@ -2244,11 +2244,13 @@ environments:
 	the_end: end, the end
 	custom: custom
 
-
+# -- Entity Potion Causes --
 entity potion causes:
 	area_effect_cloud: enter area effect cloud
 	arrow: arrow infliction
 	attack: attack
+	axolotl: axolotl
+	warden: warden
```

Comment:
i do wonder if this might cause conflicts. it might be worth registering the classinfo after entity data https://github.com/SkriptLang/Skript/blob/c4c55601a73e4fcc016d4f5f9e1bc0075912680d/src/main/java/ch/njol/skript/classes/data/BukkitClasses.java#L1530-L1534

Reply:
> When a player kills a mob that is in combat with one or more axolotls, axolotls remove [Mining Fatigue](https://minecraft.wiki/w/Mining_Fatigue) from the player, and give the player [Regeneration](https://minecraft.wiki/w/Regeneration) I for 100 game ticks (5 seconds) per axolotl in the fight, up to a duration of 2400 game ticks (2 minutes) in [Java Edition](https://minecraft.wiki/w/Java_Edition), and uncapped in [Bedrock Edition](https://minecraft.wiki/w/Bedrock_Edition).

didn't know that about axolotls, and the warden gives the darkness effect

Reply:
naming that specific cause is probably annoying, so specifying an ordering should work